habitat/tasks/rearrange/rearrange_task.py

```diff
@@ -45,6 +48,24 @@ class RearrangeTask(NavigationTask):
     def overwrite_sim_config(self, sim_config, episode):
         return merge_sim_episode_with_object_config(sim_config, episode)
 
+    def _duplicate_sensor_suite(self, sensor_suite, copy_all):
+        task_new_sensors = OrderedDict()
+        task_obs_spaces = OrderedDict()
+        for robot_idx, agent_id in enumerate(self._sim.robots_mgr.agent_names):
+            for sensor_name, sensor in sensor_suite.sensors.items():
+                if copy_all or isinstance(sensor, UsesRobotInterface):
+                    new_sensor = copy.copy(sensor)
+                    new_sensor.robot_id = robot_idx
+                    full_name = f"{agent_id}_{sensor_name}"
```

Comment:
Not sure what the format of agent_id is here. If it is an id, use `_agent{agent_id}` if it is a name, rename the variable `agent_id` to `agent_name`. 
```suggestion
                    full_name = f"{sensor_name}_agent{agent_id}"
```